tools/travis.ps1

```diff
@@ -149,7 +149,10 @@ else
         $ProgressPreference = $originalProgressPreference    
     }
 
-    $pesterParam = @{ 'binDir' = $output }
+    $pesterParam = @{ 
+        'binDir'   = $output 
+        'PassThru' = $true
+    }
```

Comment:
@anmenaga  already asked a question, why I marked these two options as mutually exclusive.  I plan on making the code paths more similar after this gets merged.